bitnami/flink/values.yaml

```diff
@@ -0,0 +1,689 @@
+  ## @param jobmanager.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param jobmanager.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param jobmanager.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param jobmanager.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 20
+    periodSeconds: 10
+    timeoutSeconds: 1
+    failureThreshold: 15
+    successThreshold: 1
+  ## @param jobmanager.customLivenessProbe Custom livenessProbe that overrides the default one
+  ##
+  customLivenessProbe: {}
+  ## @param jobmanager.customStartupProbe [object] Override default startup probe
+  ##
+  customStartupProbe: {}
+  ## @param jobmanager.customReadinessProbe [object] Override default readiness probe
+  ##
+  customReadinessProbe: {}
+  ## Apache Flink pods' resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## Minimum memory for development is 4GB and 2 CPU cores
+  ## Minimum memory for production is 8GB and 4 CPU cores
+  ## ref: http://docs.datastax.com/en/archived/flink/2.0/flink/architecture/architecturePlanningHardware_c.html
+  ##
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param jobmanager.resources.limits The resources limits for Apache Flink containers
+  ## @param jobmanager.resources.requests The requested resources for Apache Flink containers
+  ##
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 2
+    ##    memory: 4Gi
+    ##
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 2
+    ##    memory: 4Gi
+    ##
+    requests: {}
+  ## @param jobmanager.extraVolumeMounts Optionally specify extra list of additional volumeMounts for flink container
+  ##
+  extraVolumeMounts: []
+  ## Container ports to expose
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+  containerPorts:
+    ## @param jobmanager.containerPorts.rpc Port for RPC
+    ##
+    rpc: 6123
+    ## @param jobmanager.containerPorts.web Port for web UI
+    ##
+    web: 8081
+    ## @param jobmanager.containerPorts.blob Port for blob server
+    ##
+    blob: 6130
+  ## Apache Flink jobmanager.service parameters
+  ##
+  service:
+    ## @param jobmanager.service.type Apache Flink service type
+    ##
+    type: ClusterIP
+    ## Ports to expose
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ports:
+      ## @param jobmanager.service.ports.rpc Port for RPC
```

Comment:
I see references to `jobmanager.service.ports.admin`, `jobmanager.service.ports.api` and `taskmanager.service.ports.admin`. Could you check them?